### ISSUE TITLE
[Feature Request] Manga Re-read option

### ISSUE THREAD

**Issue by SunBreaker1** [Jays2Kings/tachiyomiJ2K#950]
### Why/User Benefit/User Problem
Normally when you go to read a manga for the first time, you get the prompt to "Start reading Chapter X". I personally like to re-read multiple mangas, but because the tracking is disabled after the chapter/manga is read and completed, you need to "Mark all as unread" or make the chapter "unread" by swiping left on the chapter, otherwise you need to select the chapter you want to start reading from.

### What/Requirements
This feature will essentially add a second function to the "Start reading Chapter X" button. Once the manga is completed and no longer displays the "Continue reading Chapter X" it will change to "Re-start" or "Re-read". When selected, the chapters will automatically all be marked as unread and, chapter tracking will refresh, essentially making so like the manga was just started to be read.

**Comment by CarlosEsco** [comment]
Besides the fact that its 2 buttons to mark a manga completely unread, why would you want to reset your tracker?

**Comment by SunBreaker1** [comment]
> Besides the fact that its 2 buttons to mark a manga completely unread, why would you want to reset your tracker?

When I said the thing about tracking, what I meant was; the chapter and page tracking. i.e. Chapter 3 pg.14. Sorry for the confusion, hopefully this makes sense.

**Comment by CarlosEsco** [comment]
gotcha, i thought you meant the actual 3rd tracker.   This use case is very niche, if you want to reread just mark them all urnead and hide read chapters.

**Comment by SunBreaker1** [comment]
> gotcha, i thought you meant the actual 3rd tracker. This use case is very niche, if you want to reread just mark them all urnead and hide read chapters.

Yeah, I get where your coming from, it is quite a niche feature. It would be nice to have as a feature. But if not, it's not a big deal. 
Feel free to close if this feature if you believe it is unnecessary in the grand scheme of things

**Comment by DaviDeMeo** [comment]
It would be an easy addition, but as @CarlosEsco said it's a fairly niche use case. I think users tend to re-read manga they liked a lot, we could build upon that and create a section "Favorites"(?) and include things like #805 and #630.

Right now bookmarks are (imo) underdeveloped, there's no way to quickly find bookmarked chapters and you still have to search for a specific page in them (think we you have thousands manga remembering the name of the manga 🙁).
We could add the re-read functionality to favorites manga and add an easier way to navigate bookmarked chapter with specific page marked.

In two days I'll go on vacation so i don't have time to think of how it should be implemented, when I'll come back I'd be happy to give it a shot but if it's unnecessary like @SunBreaker1 said fell free to close the issue.
